lib/mycha_spec.coffee

```diff
@@ -0,0 +1,62 @@
+async = require 'async'
```

Comment:
I originally thought about writing this in cucumber but my immediate turnoffs were
* how to require Mycha
* how to save the async return values 
Yes writing in cucumber makes the file creation easier to read, but aside from that I didn't like the complexity of getting the javascript to run. 

Mocha is awesome at unit testing, but I believe it can also be used for integration testing. To me, integration testing is just testing the entire system without stubbing anything out. I was able to do that in this test. 

The spec is very simple to me. It outputs a single file and then runs mycha and checks the err and exitCode.